packages/x-tree-view/src/internals/plugins/useTreeViewNodes/useTreeViewNodes.ts

```diff
@@ -155,7 +155,18 @@ export const useTreeViewNodes: TreeViewPlugin<UseTreeViewNodesSignature> = ({
     return childrenIds;
   };
 
+  const isItemUpdatedPreventedRef = React.useRef(false);
```

Comment:
This naming is a bit weird. Wouldn't it make sense to call this `isItemUpdatePreventedRef`?

Reply:
You are entirely right, this naming was weird
I went for `areItemUpdatesPreventedRef` since we can have multiple updates